LPS-82839 Refactor for helper method

matethurzo opened the pull request.

SpencerWoo committed:
LPS-81901 set configuration scope

aline-doleron committed:
LPS-82728 Apply overflow hidden only to Form builder

aline-doleron committed:
LPS-82728 Overwrite max-width defined by Clay media queries

aline-doleron committed:
LPS-82728 SF

aline-doleron committed:
LPS-82728 Remove extra tooltip and apply autoSave tooltip

aline-doleron committed:
LPS-82728 Change autosave message

Commit:
LPS-82464 Send new timeout parameter to Hystrix Setter

Commit:
LPS-82464 Add new timeout parameter

Commit:
LPS-82464 Remove css overriding Clay implementation

Commit:
LPS-82464 Hide pagination parameters, since it is not being used for now

Commit:
LPS-82464 Add/Fix unit tests

Commit:
LPS-82464 Extract inner class for better testability

jkappler committed:
LPS-82572 Set uuid from service context, since fragments can be stageable

jkappler committed:
LPS-82572 Listen to changes in fragment collection and entries

jkappler committed:
LPS-82572 Override methods, since it's needed for change sets

jkappler committed:
LPS-82572 Normally we use the group id from the entity, not from service context

jkappler committed:
LPS-82572 Enable staging for fragments

aline-doleron committed:
LPS-82728 Control publish link button visibility

aline-doleron committed:
LPS-82728 Remove plus button style

aline-doleron committed:
LPS-82728 Add plus button class

Commit:
LPS-82464 Add upgrade process

ZoltanTakacs committed:
APIO-198 Paths should be relative to the module

jkappler committed:
LPS-82572 System event for deleting a fragment collection and entries

jkappler committed:
LPS-82572 Auto

jkappler committed:
LPS-82572 Delete staged model were used

jkappler committed:
LPS-82572 We only want to show fragments in staging

ZoltanTakacs committed:
APIO-199 TDD

ZoltanTakacs committed:
APIO-199 Update JSONLDDocumentationMessageMapper use the correct terms for SupportedProperty

ZoltanTakacs committed:
APIO-198 Use platform independent path separator

Commit:
LPS-82572 Call through the beans otherwise the SystemEventAdvice is not going to process the annotation

Commit:
LPS-82572 That's the default

Commit:
LPS-82728 Remove tooltip for autosave message

Commit:
LPS-82728 Use correct clay classes in order to maintain management toolbar height

Commit:
LPS-82728 Update auto save message

ZoltanTakacs committed:
APIO-200 It's better to use the Advanced JSON-LD Context syntax to be able to combine external and local contexts

ZoltanTakacs committed:
APIO-200 Not needed anymore since we are using external context

ZoltanTakacs committed:
APIO-200 Can be expanded using the vocabulary

ZoltanTakacs committed:
APIO-200 Use the real relative URL for the API documentation

ZoltanTakacs committed:
APIO-200 Update documentation.json with the new changes

topolik committed:
OAUTH2-165 Align pencil icon on the right

alejandrohdezma committed:
APIO-198 No need to use `split`

alejandrohdezma committed:
APIO-200 SF

alejandrohdezma committed:
APIO-200 Duplicate

alejandrohdezma committed:
APIO-200 Simplify

jkappler committed:
LPS-82444 Use UUID from the service context

jkappler committed:
LPS-82444 Listen to changes in layout page template collection and entries

jkappler committed:
LPS-82444 Needed for changesets

jkappler committed:
LPS-82444 Fix typo

jkappler committed:
LPS-82444 Make it only visible in staging or when staging is not active

alejandrohdezma committed:
APIO-200 Create `add` vararg method in JSONObjectBuilder

Commit:
LPS-82572 Clearly separate staging and export/import controls

Commit:
LPS-82572 Apply to PDHs - set separate staging controls, and remove implied import controls

alejandrohdezma committed:
APIO-200 Javadoc SF

alejandrohdezma committed:
APIO-200 Create `arrayValue` vararg method in JSONObjectBuilder

alejandrohdezma committed:
APIO-200 Create `fields` vararg method in JSONObjectBuilder

aline-doleron committed:
LPS-82728 Change font weight to 600

aline-doleron committed:
LPS-82728 Adjust autosave message position

jkappler committed:
LPS-82444 For staging, we have to take into account all the fields when creating the new entry, so adapt accordingly

jkappler committed:
LPS-82444 Staging Handlers for asset display pages

jkappler committed:
LPS-82444 Export/Import of display pages for journal articles and for layout page template entries

jkappler committed:
LPS-82444 Add modified date when updating entries

Commit:
LPS-82728 Remove unused lang key

Commit:
LPS-82728 Build lang

brianwulbern committed:
LRQA-41736 Increment to account for wider tree

brianwulbern committed:
LRQA-41736 Reset to the right spot

p2kmgcl committed:
LPS-81654 Use visibility instead of display for hidding portlet kebab menu

Commit:
LPS-82444 Consider last publish dates

Commit:
LPS-82444 The pattern is to make the checkbox visible but disable as it's not optional to publish or not

robertoDiaz committed:
LPS-82801 We should not pass to the url the previous keywords parameter for the search URL. It will result in a duplication that could break the logic, since in this form the keyword parameter should be send using the input.

p2kmgcl committed:
LPS-81654 Remove extra spacing on barebone portlet skin

Commit:
LPS-82668 Implement containsKey

to avoid `java.util.ResourceBundle.containsKey` method that invokes
parent's `handleKeySet`

Commit:
LPS-82668 Cache the final ResourceBundleLoader

to minimize lookups

xbrianlee committed:
LRQA-36852 Re-enable tc Server portal log assertor test

Commit:
LPS-82338 Return the title of the definition considering Locale

rafaprax committed:
LPS-82728 Update unit test

rafaprax committed:
LPS-82728 Update poshi test

lesliewong92 committed:
LRQA-41850 Semantic versioning checks are not needed for test modules

lesliewong92 committed:
LRQA-41850 Specify modules includes/excludes for private/public branches

lesliewong92 committed:
LRQA-41850 Modify modules inclusion/exclusion properties to match new logic

jkappler committed:
LPS-82572 Consider last publish dates

jkappler committed:
LPS-82572 Baseline

alejandrohdezma committed:
APIO-194 Use latest service.tracker.collections version

brianwulbern committed:
LRQA-41736 Go to the appropriate lexicon item

brianwulbern committed:
LRQA-41736 Don't need to close modal if navigating to page afterwards

hhuijser committed:
LPS-82828 Deprecated as of 6.0.0 or earlier

hhuijser committed:
LPS-82828 Deprecated as of 6.1.0

hhuijser committed:
LPS-82828 Deprecated as of 6.2.0

hhuijser committed:
LPS-82828 Deprecated as of 7.0.0

hhuijser committed:
LPS-82828 Deprecated as of 7.1.0

hhuijser committed:
LPS-82828 ant format-javadoc

hhuijser committed:
LPS-82828 Apply to .ftl

dantewang committed:
See https://github.com/shuyangzhou/liferay-portal/pull/6370#issue-196985792 Revert "LPS-82545 I think this is a better spot?"

This reverts commit 945a77f9002f52e840b08f6404e6e63a3e033adf.

hhuijser committed:
LPS-82828 ant build-services

hhuijser committed:
LPS-82828 Disable JavaDeprecatedJavadocCheck for now. Will update logic in later commit

hhuijser committed:
LPS-82828 Manually change cases in trash

hhuijser committed:
LPS-82828 Manually change missed ones

hhuijser committed:
LPS-82828 SF

hhuijser committed:
LPS-82828 ant build-iw

brianwulbern committed:
LRQA-41736 Pass in the page

brianwulbern committed:
LRQA-41736 Use current navigation method

brianwulbern committed:
LRQA-41736 Logo now embedded within Advanced tab

brianwulbern committed:
LRQA-41736 Update Windows Open button screenshot

pavel-savinov committed:
LPS-82470 Return layout page template as staged model by classPK to reuse it in fragment entry link staged model data handler

pavel-savinov committed:
LPS-82470 Export/import associated staged model entities

brianwulbern committed:
LRQA-36536 Remove due to outdated functionality

pavel-savinov committed:
LPS-82470 Import referenced layout page template collection before getting new primary keys map

ealonso committed:
LPS-82819 Check update permissions

pavel-savinov committed:
LPS-81954 Use new method to take in account selected site navigation menu type

marcoscv committed:
LPS-82721 - Borderless / barebone decorators styles

ealonso committed:
LPS-82444 SF

pavel-savinov committed:
LPS-82470 Fix integration test - create layout to be sure that fragment entry link has correct class pk

boton committed:
LPS-81488 Add clickeable area with empty editable field

Commit:
LPS-82754 radio value is text not boolean - templates referenced contents radio button lost their values

Commit:
LPS-81901 SF

Commit:
LPS-82564 Even though portlet is hidden create the control panel entry to be able to control access to the portlet's features

Commit:
LPS-81901 Missed some SSO configurations

Commit:
LPS-81901 OpenIdConnectProviderConfiguration is actually System scope

jkappler committed:
LPS-82572 Auto

jkappler committed:
LPS-82572 Use new method

jkappler committed:
LPS-82444 We need to update the display page associated, mostly for listening for changes for staging

jkappler committed:
LPS-82444 Baseline

GergoMathe committed:
LPS-81816 Use ExportImportClassedModelUtil to avoid separate logic for DL

Commit:
LPS-82839 Fix for control panel

Commit:
LPS-82839 Refactor for helper method

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

pavel-savinov committed:
LPS-82766 Adjust styles to make layouts tree look the same as navigation menu tree

Commit:
LPS-82754 SF

jpince committed:
Revert "LRQA-41869 Don't include in the build due to websphere compile error"

This reverts commit 589fa1d1671cc99e612901519c3e3b234abab013.

jpince committed:
LRQA-41869 Exclude websphere due to compile error

jpince committed:
Revert "LRQA-41695 Remove failing test from acceptance" now that
LPS-82797 is merged

This reverts commit b8c23a8278001e04c5c04334b2b261acd0b2ec16.

p2kmgcl committed:
LPS-81654 Add 7.0 colors

liferay-continuous-integration committed:
LPS-77699 Update translations

Commit:
LPS-80825 Updates application id and event names for FileEntry downloads

p2kmgcl committed:
LPS-81654 Update colors for 7.1

Commit:
LPS-82839 add method to retrieve control panel layout's plid

Commit:
LPS-82839 generated

Commit:
LPS-82839 Baseline

codyhoag committed:
LPS-82130 Edit changelogs

topolik committed:
LPS-82849 Disable telnet to gogo shell in production, keep for upgrade and developers

jbalsas committed:
LPS-82721 SF

topolik committed:
LPS-75864 Check CONFIGURE_PORTLETS only if the layout actually supports portlets.

It doesn't make sense to check CONFIGURE_PORTLETS for URL, Link to Page or Control Panel layout(s).

Commit:
LPS-80947 Rename Liferay Invite to Liferay Invite Members

Commit:
LPS-81901 Semver

brianchandotcom committed:
LPS-82464 Rename

rafaprax committed:
LPS-82648 Delete structures after delete the form's entries and use the right delete method

brianchandotcom committed:
LPS-82849 As it appears in portal.properties, Hugo SF, this is for portal-ext or all portal-*.properties

brianchandotcom committed:
Revert "LRQA-41869 Exclude websphere due to compile error"

This reverts commit fd0c679ba25f45c12b48332f9c38a9d3942bf438.

brianchandotcom committed:
LPS-82754 Optimize for the human brain

brianchandotcom committed:
LPS-82828 prep next

Commit:
LPS-82850 Deploy only, no need to include it

Commit:
LPS-82850 Prevent wrong versions from being imported

brianchandotcom committed:
LPS-82828 prep next

brianchandotcom committed:
LRQA-41850 prep next

mtambara committed:
LPS-82333 Make sure to always return a full locale, not just the language

vicnate5 committed:
LRQA-41340 Re-add ignore for LPS-81732

pyoo47 committed:
LRQA-41133 Avoid ConcurrentModificationException

petershin committed:
LPS-0 Update liferay-versions to '7.1.0+'

jpince committed:
LRQA-41876 Ignore flaky test

marcellustavares committed:
LPS-82628 Update JS client to point to production gateway

shuyangzhou committed:
LPS-82460 Register the logger through system property

Ithildir committed:
LPS-82857 Find the oldest commit in a branch

Ithildir committed:
LPS-67653 Make methods private to reduce API surface

Ithildir committed:
LPS-82857 Export package

Ithildir committed:
LPS-82857 Add packageinfo

Ithildir committed:
LPS-82857 Enforce semantic versioning

Ithildir committed:
LPS-67653 Move unexported package to "internal"

Ithildir committed:
LPS-82857 Semver gradle-plugins-change-log-builder

Ithildir committed:
LPS-67653 Update changelog

Ithildir committed:
LPS-82857 Update changelog

brianchandotcom committed:
LPS-82460 Sort

Ithildir committed:
LPS-82857 Use latest gradle-plugins-change-log-builder

brianchandotcom committed:
LRQA-41133 prep next

marcellustavares committed:
LPS-82627 Update Java clients to point to production gateway

petershin committed:
LPS-0 Update theme version to '7.1.0+'

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/analytics/.gitrepo'.

Preston-Crary committed:
LPS-82860 Do not create the layout just to delete it.

codyhoag committed:
LPS-82849 New breaking change for disabling Gogo Shell via telnet

brianchiu committed:
LPS-81423 Location assertion not needed, just wait for page reload

lesliewong92 committed:
LRQA-41133 Use repository name instead of repository directory path

michaelhashimoto committed:
LRQA-41877 Only deploy a new source format jar when the module is available

jkappler committed:
LPS-82444 Auto SF

brianchandotcom committed:
apps/portal-security-sso/portal-security-sso-ntlm-api packageinfo

brianchandotcom committed:
portal-security-sso-openid-api packageinfo

brianchandotcom committed:
com.liferay.portal.security.sso.openid.connect.configuration packageinfo

brianchandotcom committed:
com.liferay.portal.security.sso.opensso.configuration.OpenSSOConfiguration packageinfo

petershin committed:
LPS-82828 Semver portal-kernel

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/apio-architect/.gitrepo'.

brianchandotcom committed:
LRQA-41133 prep next

shuyangzhou committed:
LPS-82333 SF

Commit:
LPS-82850 Disable the failing tests for the moment

They only fail when ran from the suite but work when run separately, so
the feature seem to work but there exists some interaction in the test
suite.

brianchandotcom committed:
LPS-82444 Hugo SF

brianchandotcom committed:
LPS-82444 SF

brianchandotcom committed:
LPS-82828 prep next

brianchandotcom committed:
LPS-82857 prep next

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 2209fccc19cdc1d94866e8c04f72405f0f4a69d6

petershin committed:
LPS-0 Update 'com.liferay.portal.kernel.util' version
